Source/ACE/Managers/GuidManager.cs

```diff
@@ -127,22 +127,26 @@ public uint Alloc()
         private static uint staticObject = staticObjectMax;
         */
 
+        public static uint GeneratorMin { get; } = ObjectGuid.GeneratorMin;
```

Comment:
If we're moving these to ObjectGuid (which I'm okay with), then I would remove the constants from here, and just use the ObjectGuid constants instead.